test/jdk/java/util/regex/whitebox/GraphemeTest.java

```diff
@@ -23,24 +23,33 @@
 
 /*
  * @test
- * @bug 7071819 8221431 8239383 8273430
+ * @bug 7071819 8221431 8239383 8273430 8291660
  * @summary tests Unicode Extended Grapheme support
  * @library /lib/testlibrary/java/lang
- * @build java.base/java.util.regex.GraphemeTestAccessor
- * @run testng GraphemeTest
+ * @compile --add-exports java.base/jdk.internal.util.regex=ALL-UNNAMED GraphemeTest.java
+ * @run testng/othervm --add-exports java.base/jdk.internal.util.regex=ALL-UNNAMED --add-opens java.base/jdk.internal.util.regex=ALL-UNNAMED GraphemeTest
```

Comment:
Can you use the `@modules` directive to export+open the internal module to the test?

Reply:
Good point. Will use `@modules` tag.